From 5da7bf929b3105a0cf5fec52834821e21f422861 Mon Sep 17 00:00:00 2001
From: Lily Zhang <r58066@freescale.com>
Date: Wed, 27 Aug 2008 21:54:11 +0800
Subject: [PATCH 047/505] ENGR00089418 Remove MX51 unsupported modules from
 Makefile

Remove MX51 unsupported modules from test Makefile.
Correct some wrong platform names

Signed-off-by: Lily Zhang <r58066@freescale.com>
---
 test/demo/Makefile               | 2 +-
 test/max8660_pmic_test/Makefile  | 6 +++++-
 test/mxc_asrc_test/Makefile      | 2 +-
 test/mxc_firi_test/Makefile      | 2 +-
 test/mxc_hdd_test/autorun-ata.sh | 2 +-
 test/mxc_i2c_slave_test/Makefile | 2 +-
 test/mxc_ipc_test/Makefile       | 2 +-
 test/mxc_mlb_test/Makefile       | 2 +-
 test/mxc_mpeg4_enc_test/Makefile | 2 +-
 test/mxc_mpeg4_test/Makefile     | 2 +-
 test/mxc_mu_test/Makefile        | 2 +-
 test/mxc_owire_test/Makefile     | 2 +-
 test/mxc_pf_test/Makefile        | 2 +-
 test/mxc_pmic_test/Makefile      | 2 +-
 test/mxc_rng_test/Makefile       | 2 +-
 test/mxc_scc2/Makefile           | 2 +-
 test/mxc_security_test/Makefile  | 2 +-
 test/mxc_udma_test/Makefile      | 2 +-
 18 files changed, 22 insertions(+), 18 deletions(-)

diff --git a/test/demo/Makefile b/test/demo/Makefile
index b89cf2b..4b36448 100644
--- a/test/demo/Makefile
+++ b/test/demo/Makefile
@@ -3,7 +3,7 @@
 #
 
 # list of platforms which did not want this test case
-EXCLUDE_LIST:= IMX35_3STACK
+EXCLUDE_LIST:= "IMX35_3STACK IMX51_3STACK"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 CONFIG_DEMO_DPTCHI=y
diff --git a/test/max8660_pmic_test/Makefile b/test/max8660_pmic_test/Makefile
index 1063323..0786bfe 100644
--- a/test/max8660_pmic_test/Makefile
+++ b/test/max8660_pmic_test/Makefile
@@ -1,11 +1,15 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX37_3STACK IMX31_3STACK IMX31_ADS IMX27_3STACK IMX27_ADS"
+EXCLUDE_LIST:="IMX51_3STACK IMX37_3STACK IMX31_3STACK IMX31ADS IMX27_3STACK IMX27ADS"
 
 
 LTP_DIR = $(TOPDIR)/test/mxc_pmic_test/
 LOADLIBES+= -L$(LTP_DIR)lib -lltp
 
+ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = autorun-mcu_pmic.sh libltp.a
+else
+OBJS =
+endif
 
 CFLAGS+= -Wall -I$(LTP_DIR)/include
 CFLAGS+= $(INC) -I$(LINUXPATH)/include/ -D__CVS_TEST
diff --git a/test/mxc_asrc_test/Makefile b/test/mxc_asrc_test/Makefile
index a815078..739d3bd 100644
--- a/test/mxc_asrc_test/Makefile
+++ b/test/mxc_asrc_test/Makefile
@@ -2,7 +2,7 @@
 # Makefile for OSS sound driver test app.
 #
 # list of platforms which did not want this test case
-EXCLUDE_LIST:= IMX21ADS IMX27ADS IMX31ADS IMX32ADS IMX37_3STACK
+EXCLUDE_LIST:= IMX21ADS IMX27ADS IMX31ADS IMX32ADS IMX37_3STACK IMX51_3STACK
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = \
diff --git a/test/mxc_firi_test/Makefile b/test/mxc_firi_test/Makefile
index 3ec8d2f..06146a6 100644
--- a/test/mxc_firi_test/Makefile
+++ b/test/mxc_firi_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="MXC27520EVB MXC27530EVB IMX27ADS MXC30020EVB MXC30031ADS IMX37_3STACK IMX35_3STACK"
+EXCLUDE_LIST:="MXC27520EVB MXC27530EVB IMX27ADS MXC30020EVB MXC30031ADS IMX37_3STACK IMX35_3STACK IMX51_3STACK"
 
 # list of platforms which do not have either SIR or FIR
 IR_EXCLUDE_LIST:=
diff --git a/test/mxc_hdd_test/autorun-ata.sh b/test/mxc_hdd_test/autorun-ata.sh
index ef66b77..3d9faea 100755
--- a/test/mxc_hdd_test/autorun-ata.sh
+++ b/test/mxc_hdd_test/autorun-ata.sh
@@ -49,7 +49,7 @@ then
 	modprobe ide_disk
 	sleep 1
 else
-	echo "IMX35_3STACK"
+	echo "$(platform)"
 	modprobe pata_fsl
 	sleep 5
 fi
diff --git a/test/mxc_i2c_slave_test/Makefile b/test/mxc_i2c_slave_test/Makefile
index 2eaf205..0cdc2fc 100644
--- a/test/mxc_i2c_slave_test/Makefile
+++ b/test/mxc_i2c_slave_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:=IMX31ADS IMX27ADS IMX37_3STACK IMX31_3STACK
+EXCLUDE_LIST:=IMX31ADS IMX27ADS IMX37_3STACK IMX31_3STACK IMX51_3STACK
 
 CFLAGS += -lpthread
 
diff --git a/test/mxc_ipc_test/Makefile b/test/mxc_ipc_test/Makefile
index 262a4dc..bd0746e 100644
--- a/test/mxc_ipc_test/Makefile
+++ b/test/mxc_ipc_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX21ADS IMX27ADS IMX31ADS MXC91131EVB IMX37_3STACK IMX35_3STACK"
+EXCLUDE_LIST:="IMX21ADS IMX27ADS IMX31ADS MXC91131EVB IMX37_3STACK IMX35_3STACK IMX51_3STACK"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/mxc_ipc_test.out
diff --git a/test/mxc_mlb_test/Makefile b/test/mxc_mlb_test/Makefile
index 365a6d4..d9889bb 100644
--- a/test/mxc_mlb_test/Makefile
+++ b/test/mxc_mlb_test/Makefile
@@ -2,7 +2,7 @@
 # Makefile for MediaLB driver test app.
 #
 # list of platforms which did not want this test case
-EXCLUDE_LIST:= IMX21ADS IMX27ADS IMX31ADS IMX32ADS IMX37_3STACK
+EXCLUDE_LIST:= IMX21ADS IMX27ADS IMX31ADS IMX32ADS IMX37_3STACK IMX51_3STACK
 
 INC += -I$(LINUXPATH)/include
 
diff --git a/test/mxc_mpeg4_enc_test/Makefile b/test/mxc_mpeg4_enc_test/Makefile
index e7e2999..6e26bc4 100644
--- a/test/mxc_mpeg4_enc_test/Makefile
+++ b/test/mxc_mpeg4_enc_test/Makefile
@@ -5,7 +5,7 @@ OBJDIR=$(TOPDIR)/platform/$(PLATFORM)/$(MODULE_NAME)
 CFLAGS += -I . -lpthread
 
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX21ADS IMX27ADS IMX37_3STACK IMX35_3STACK"
+EXCLUDE_LIST:="IMX21ADS IMX27ADS IMX37_3STACK IMX35_3STACK IMX51_3STACK"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 ifeq "$(PLATFORM)" "MXC30030EVB"
diff --git a/test/mxc_mpeg4_test/Makefile b/test/mxc_mpeg4_test/Makefile
index 0d1d75e..30e131a 100644
--- a/test/mxc_mpeg4_test/Makefile
+++ b/test/mxc_mpeg4_test/Makefile
@@ -8,7 +8,7 @@ LIBS = $(SRCDIR)/libmpeg4_enc_ELINUX.a $(SRCDIR)/libmpeg4_dec_ELINUX.a -lm -lc -
 CFLAGS += -DNDEBUG -DPROFILE -msoft-float -mfpu=vfp
 
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX21ADS IMX27ADS"
+EXCLUDE_LIST:="IMX21ADS IMX27ADS IMX51_3STACK"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 EXES = \
diff --git a/test/mxc_mu_test/Makefile b/test/mxc_mu_test/Makefile
index 2bec6f6..b9df166 100644
--- a/test/mxc_mu_test/Makefile
+++ b/test/mxc_mu_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX27ADS IMX31ADS IMX37_3STACK IMX35_3STACK"
+EXCLUDE_LIST:="IMX27ADS IMX31ADS IMX37_3STACK IMX35_3STACK IMX51_3STACK"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = \
diff --git a/test/mxc_owire_test/Makefile b/test/mxc_owire_test/Makefile
index 88bf29e..9a584da 100644
--- a/test/mxc_owire_test/Makefile
+++ b/test/mxc_owire_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX31_3STACK IMX37_3STACK IMX35_3STACK"
+EXCLUDE_LIST:="IMX31_3STACK IMX37_3STACK IMX35_3STACK IMX51_3STACK"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/autorun-owire.sh
diff --git a/test/mxc_pf_test/Makefile b/test/mxc_pf_test/Makefile
index 9e893f2..9f87ad3 100644
--- a/test/mxc_pf_test/Makefile
+++ b/test/mxc_pf_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX27ADS IMX37_3STACK"
+EXCLUDE_LIST:="IMX27ADS IMX37_3STACK IMX51_3STACK"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/autorun-pf.sh $(OBJDIR)/mxc_pf_test.out
diff --git a/test/mxc_pmic_test/Makefile b/test/mxc_pmic_test/Makefile
index a0b508b..7a34a59 100644
--- a/test/mxc_pmic_test/Makefile
+++ b/test/mxc_pmic_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX37_3STACK IMX35_3STACK"
+EXCLUDE_LIST:="IMX37_3STACK IMX35_3STACK IMX51_3STACK"
 
 LTP_DIR = ../
 LOADLIBES+= -L$(LTP_DIR)lib -lltp -lpthread -lm
diff --git a/test/mxc_rng_test/Makefile b/test/mxc_rng_test/Makefile
index 81e5f5c..b90d0b9 100644
--- a/test/mxc_rng_test/Makefile
+++ b/test/mxc_rng_test/Makefile
@@ -5,7 +5,7 @@
 #****************************************************************************/
 
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX27ADS IMX27EVB MXC27530EVB MXC30030ADS MXC30030EVB MXC30031ADS"
+EXCLUDE_LIST:="IMX27ADS IMX27EVB MXC27530EVB MXC30030ADS MXC30030EVB MXC30031ADS IMX51_3STACK"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = apitest rng_test move
diff --git a/test/mxc_scc2/Makefile b/test/mxc_scc2/Makefile
index 88a76cb..89e9477 100644
--- a/test/mxc_scc2/Makefile
+++ b/test/mxc_scc2/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX21ADS IMX27_ADS IMX31_ADS IMX31_3STACK IMX35_3STACK"
+EXCLUDE_LIST:="IMX21ADS IMX27ADS IMX31ADS IMX31_3STACK IMX35_3STACK"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/scc2_test.out $(OBJDIR)/autorun-scc2.sh
diff --git a/test/mxc_security_test/Makefile b/test/mxc_security_test/Makefile
index 3e50f22..94c9651 100644
--- a/test/mxc_security_test/Makefile
+++ b/test/mxc_security_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX37_3STACK IMX35_3STACK"
+EXCLUDE_LIST:="IMX37_3STACK IMX35_3STACK IMX51_3STACK"
 
 # list of platforms that have hacc
 HACC_LIST:=MXC91131EVB
diff --git a/test/mxc_udma_test/Makefile b/test/mxc_udma_test/Makefile
index b07886c..9e19394 100644
--- a/test/mxc_udma_test/Makefile
+++ b/test/mxc_udma_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:= IMX31ADS I30030EVB I30030ADS MXC30030EVB MXC30030ADS MXC91131EVB MXC27530EVB IMX35_3STACK
+EXCLUDE_LIST:= IMX31ADS I30030EVB I30030ADS MXC30030EVB MXC30030ADS MXC91131EVB MXC27530EVB IMX35_3STACK IMX51_3STACK
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/mxc_udma_test.out
-- 
1.8.0

